content/en/docs/concepts/overview/object-management-kubectl/kustomization.md

```diff
@@ -0,0 +1,752 @@
+---
+title: Declarative Management of Kubernetes Objects Using Kustomization
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+[Kustomize](https://github.com/kubernetes-sigs/kustomize) is a standalone tool
+to customize Kubernetes objects
+through a [kusotmization file](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/kustomization.yaml). 
+Since 1.14, Kubectl also
+supports the management of Kubernetes objects using a kustomization file.
+When dealing with Resources or objects from kustomization, the flag
+`--kusotmize` or `-k` need to be used in Kubectl commands.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Overview of Kustomize
+Kustomize is a tool for customizing Kubernetes configuration. 
+It offers simple tooling to manage configuration files for your applications with features from aspects as
+* generating resources from other sources
```

Comment:
nit: Add a line before starting the itemized list so that the list renders
nit: Do you need to repeat this info below? Possibly remove from one location.

Reply:
updated